pkg/environment/environment.go

```diff
@@ -65,33 +69,24 @@ func MustGetKrewPathsFromEnvs(envs []string) KrewPaths {
 		Index:    filepath.Join(base, "index"),
 		Install:  filepath.Join(base, "store"),
 		Download: filepath.Join(os.TempDir(), "krew"),
+		Bin:      filepath.Join(base, "bin"),
 	}
 }
 
-func getKubectlPluginsPath(envs []string) string {
-	envvars := parseEnvs(envs)
-
-	// Look for "${KUBECTL_PLUGINS_PATH}"
-	if path, ok := envvars["KUBECTL_PLUGINS_PATH"]; ok && path != "" {
```

Comment:
Does the implementation completely ignore KUBECTL_PLUGINS_PATH now? (I'm just trying to double-check.) The 1.12 release branch docs don't seem to be updated. https://github.com/kubernetes/website/blob/release-1.12/content/en/docs/tasks/extend-kubectl/kubectl-plugins.md